openfisca_france/parameters/prestations_sociales/solidarite_insertion/minima_sociaux/accident_travail/rente/taux/indemnite_accident_travail.yaml

```diff
@@ -1,72 +1,103 @@
 description: Montant de l'indemnité en capital attribuée à la victime d'un accident du travail
 brackets:
 - threshold:
-    2016-04-01:
+    2002-03-30:
```

Comment:
Si je ne me trompe pas, et au vu de l'article [L434-16](https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000031688385/), il semble que la date d'application de la nouvelle valeur dans les calculs soit le 1er avril. Il faudrait donc changer les dates pour `2002-04-01`
Est-ce que @pzuldp , grand expert des dates, tu confirmes ce raisonnement ? :)

Reply:
oui, ça me semble juste, il faut remplacer avec le 1er avril.
le truc plus embêtant c'est qu'il y a normalement une revalo _chaque_ 1er avril, comme indiqué dans l'article que tu mentionnes... Ainsi, ici : https://www.cleiss.fr/actu/breves.php on voit que dans les actualités de chaque mois d'avril, il y a une circulaire qui indique le taux de revalorisation (ce taux est égal à la prévision d'inflation hors pour l'année en cours, auquel on ajoute le différentiel entre la prévision et la réalisation de l'année dernière - pour corriger incrémentalement les erreurs de prédiction).
de même, au 1er avril 2022, les valeurs sont différentes de celles indiquées au 1er avril 2016 (https://www.service-public.fr/particuliers/vosdroits/F14840), ce qui est normal car il y a eu 6 ans d'inflation depuis...
mais attention, quand l'inflation est négative, il n'y a pas de revalorisation (cf la circulaire pour le 1er avril 2015, par exemple : http://www.legislation.cnav.fr/Documents/circulaire_cnav_2015_23_16042015.pdf).

Reply:
Bonjour @pzuldp @Sasha-Laniece merci beaucoup pour vos contributions ! :)

Reply:
@AhmedZemzami pour la date c'est OK, et tes changements sont bons.


En revanche Paul soulève un point intéressant: il faut effectuer la revalorisation tous les ans. S'il semble que trouver le taux de revalorisation soit compliqué, tu peux peut-être t'en sortir facilement: c'est probablement le taux utilisé pour revaloriser la BMAF.

Il faut donc:
- [ ] Chercher dans la loi les références qui indiquent que les modalités de calcul du taux de revalorisation de la BMAF sont les mêmes que pour le taux utilisé ici
- [ ] Calculer toutes les valeurs entre 2002 et 2022 et les mettre dans le paramètre
- [ ] Mettre dans les notes les taux utilisés (pour pouvoir se relire par la suite)

À ton excel ! ;)

Reply:
C'est noté, je vais faire comme ça merci ! ;)

Reply:
https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000031781092/ ---> Du coup les indemnités sont calculé comme pour toutes les prestations sociales hors pensions de vieillesse.

Reply:
Du coup il me semble que le calcul du taux de revalorisation de la BMAF et pour les indemnités accident du travail est le même

Reply:
2002 | 341.87 |  
-- | -- | --
2003 | 341.88 | 0.0029%
2004 | 353.59 | 3.42%
2005 | 361.37 | 2.2%
2006 | 367.87 | 1.80%
2007 | 374.12 | 1.70%
2008 | 377.86 | 1.00%
2009 | 389.2 | 3.00%
2010 |   |  
2011 | 395.04 |  
2012 | 399 | 1.00%
2013 | 403.79 | 1.20%
2014 | 406.21 | 0.59%
2015 |   |  
2016 |   | 0.1%
2017 |   | 0.3%
2018 |   | 1.00%
2019 |   | 0.3%
2020 |   | 0.3%
2021 |   | 0.1%

Voici les valorisations de la BMAF pour chaque année depuis 2002 ( sauf 2015 car pas de value). Je pourrais appliquer ce taux aux différentes tranches d'indemnités accident du travail depuis 2002

Reply:
Yes c'est ca. Pourquoi pas de value en 2015 ? Est-ce parce que l'inflation etait <1 ? Auquel cas on prend 1

Reply:
Je viens de vérifier et la bmaf a pas changé entre 2014 et 2015 ( on voit cela dans les circulaires 2014-15) du coup je suis pas sur que l'on puisse prendre 1

Reply:
@Sasha-Laniece oui c'est ça  http://www.legislation.cnav.fr/Documents/circulaire_cnav_2015_23_16042015.pdf

@AhmedZemzami Sasha et toi êtes d'accord, Sasha dit "prendre 1" au sens de multiplier la valeur de 2014 par 1 pour obtenir la valeur de 2015 (multiplier par 1 + t, où t est le taux utilisé, et où ici, t = 0), d'où on obtient bien, comme tu dis, la même valeur en 2014 et 2015.
Sasha veut peut-être faire apparaître explicitement que le taux est de 0 pour 2015 pour qu'il n'y ait pas d'ambiguïté possible (pour que personne ne se repose la question de savoir si la valeur est manquante ou s'il n'y a effectivement pas d'inflation cette année là). C'est ça ?

Reply:
Merci @pzuldp, effectivement @Sasha-Laniece a complètement raison, je vais donc rajouter le 1 pour 2015 ;)